web-app/helpers/certificate-designs.ts

```diff
@@ -4,17 +4,17 @@ import { Canvas, registerFont, loadImage } from 'canvas';
 // TODO: Update this section:
 export const width = 1080; // width of the image
 export const height = 1080; // height of the image
-const fontFile = './fonts/Manrope-VariableFont_wght.ttf';
-const fontFamily = 'Manrope, Sans Serif';
-const fontFile2 = './fonts/DMMono-Medium.ttf';
-const fontFamily2 = 'DM Mono, monospace';
+const manropeFontFile = './fonts/Manrope-VariableFont_wght.ttf';
+const manropeFontFamily = 'Manrope, Sans Serif';
+const monoFontFile = './fonts/DMMono-Medium.ttf';
+const monoFontFamily = 'DM Mono, monospace';
 
-registerFont(fontFile, { family: fontFamily });
-registerFont(fontFile2, { family: fontFamily2 });
+registerFont(manropeFontFile, { family: manropeFontFamily });
+registerFont(monoFontFile, { family: monoFontFamily });
 
 function getBaseContext(canvas: Canvas) {
   const context = canvas.getContext('2d');
-  context.textAlign = 'left';
+  context.textAlign = 'center';
```

Comment:
I tried to do it but it doesn't solve the problem I guess, just made me more confused :D 
Here is a video, I try to explain myself here https://www.loom.com/share/240aee2b870849cba6da63355b8cdc0b

Reply:
@ryancwalsh wdyt?
https://www.loom.com/share/edb7451caf52441cb26fb2ce9cb41ff5

Reply:
@norrec99 Thanks for the video. I don't think we're understanding each other, though. If you have time, maybe you can ask someone else on T3 to look at our 2 recent videos and this comment and see whether they can help.  

At the beginning of your video, I thought you were saying that the docs show that the top left of a rectangle is what the X, Y position determines. But then it looked like you ignored that fact and tried to calculate positions other than the top left of each rectangle.

I really think that if you position each rectangle first appropriately, then aligning the text _within_ each rectangle will be as simple as setting "left" or "center".

For example, look at your video at 2:01 where you show Dan's design. (It would be helpful if you paste the link to that here in your comment.) 

Let's look at the program name rectangle. My guess is that this is what would work:
X = 65
Y = 65 + 65 + 40 + 49 + 85 + 261 + 85
text align = center